src/pybind/mgr/dashboard/frontend/src/styles.scss

```diff
@@ -1,2 +1,293 @@
 /* You can add global styles to this file, and also import other style files */
-@import './openattic-theme.scss';
+
+@import 'defaults';
+$fa-font-path: "../node_modules/fork-awesome/fonts";
+@import "../node_modules/fork-awesome/scss/fork-awesome";
+$font_family_1: 'ForkAwesome';
+
+/* Basics */
+html {
+  background-color: $color-app-bg;
+}
+html,
+body {
+  width: 100%;
+  height: 100%;
+  font-size: 12px;
+}
+a {
+  color: $color-links;
+  cursor: pointer;
+}
+a:hover,
+a:focus{
+  color: $color-links-focus
+}
+h1 {
+  letter-spacing: -1px;
+  font-size: 2em;
+}
+h2 {
+  letter-spacing: -1px;
+  font-size: 1.833em;
+}
+h3{
+  display: block;
+  font-size: 1.583em;
+  font-weight: 400;
+}
+h4{
+  font-size: 1.5em;
+  line-height: normal
+}
+option {
+  font-weight: normal;
+  font-style: normal;
+}
+/*For Opera browser*/
```

Comment:
It  seems Opera is no longer a problem. I'm guessing since they are now using the same render as Chrome.
This can be replaced by:
```
/* For awesome-bootstrap-checkbox */
.checkbox input[type='checkbox']:checked + label:after {
  font-family: $font_family_1;
}
/*******/
```

Also, please don't use tabs, we use 2 spaces for indentation.

Reply:
Ack